src/workerd/jsg/jsg.h

```diff
@@ -2063,6 +2103,75 @@ class Lock {
   ContextScope enterContextScope(v8::Local<v8::Context> context);
   // Ensures that we are currently within the scope of the given v8::Context
 
+  // ====================================================================================
+  JsValue undefined() KJ_WARN_UNUSED_RESULT;
+  JsValue null() KJ_WARN_UNUSED_RESULT;
+  JsBoolean boolean(bool val) KJ_WARN_UNUSED_RESULT;
+  JsNumber num(double) KJ_WARN_UNUSED_RESULT;
+  JsNumber num(float) KJ_WARN_UNUSED_RESULT;
+  JsInt32 num(int8_t) KJ_WARN_UNUSED_RESULT;
+  JsInt32 num(int16_t) KJ_WARN_UNUSED_RESULT;
+  JsInt32 num(int32_t) KJ_WARN_UNUSED_RESULT;
+  JsUint32 num(uint8_t) KJ_WARN_UNUSED_RESULT;
+  JsUint32 num(uint16_t) KJ_WARN_UNUSED_RESULT;
+  JsUint32 num(uint32_t) KJ_WARN_UNUSED_RESULT;
+  JsBigInt bigInt(int64_t) KJ_WARN_UNUSED_RESULT;
+  JsBigInt bigInt(uint64_t) KJ_WARN_UNUSED_RESULT;
+  JsString str() KJ_WARN_UNUSED_RESULT;
+  JsString str(kj::ArrayPtr<char16_t>) KJ_WARN_UNUSED_RESULT;
+  JsString str(kj::ArrayPtr<const char16_t>) KJ_WARN_UNUSED_RESULT;
+  JsString str(kj::ArrayPtr<uint16_t>) KJ_WARN_UNUSED_RESULT;
+  JsString str(kj::ArrayPtr<const char>) KJ_WARN_UNUSED_RESULT;
+  JsString str(kj::ArrayPtr<char>) KJ_WARN_UNUSED_RESULT;
```

Comment:
I think you can delete the non-const overloads here. If someone passes `ArrayPtr<char>` it'll match `ArrayPtr<const char>` implicitly.

Reply:
done